editor/utils/map.ts

```diff
@@ -67,3 +68,18 @@ export const useCurrentBounds = (
       [bounds[2], bounds[3]],
     ];
   }, [geojson]);
+
+/**
+ * Returns the URL for the basemap type depending on app theme
+ */
+export const getBasemapURL = (
```

Comment:
probably not much of a concern because this is a super simple computation, but maybe a nice touch to convert this to a `useBasemap` hook?